galaxy_ng/app/webserver_snippets/nginx.conf

```diff
@@ -0,0 +1,3 @@
+location / {
+    alias /var/lib/pulp/assets/galaxy_ng/;
```

Comment:
It works only with the change in https://github.com/pulp/pulp_installer/pull/268/files